ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsPruning.scala

```diff
@@ -88,7 +88,7 @@ private[backend] trait StorageBackendTestsPruning extends Matchers with StorageB
       contractId = "#1",
       signatory = someParty,
     )
-    val createFilter1 = DbDto.CreateFilter(1L, someTemplateId.toString, "signatory")
+    val createFilter1 = DbDto.CreateFilter(1L, someTemplateId.toString, "party")
```

Comment:
Why is this rename required?

Reply:
This implementation has no filter table, and is getting the filtering info directly from events. So if the test needed to be changed to align the fixtures: same information in create-filters and create-events.
There is a TODO though to add improvement of filter tables (that would be nested maps as indexes in the in memory implementation), because that would be more performant for the in memory as well, and also it would be more conformant as well. (not sure it is needed though)

Reply:
Ah OK. Thanks.